RecoBTag/Configuration/python/RecoBTag_EventContent_cff.py

```diff
@@ -32,6 +32,9 @@
         'keep *_pfDeepCSVJetTags_*_*',
         # DeepCMVA
         'keep *_pfDeepCMVAJetTags_*_*',
+        # DeepFlavour (new)
+        'keep *_pfDeepFlavourTagInfos_*_*',
+        'keep *_pfDeepFlavourJetTags_*_*',
```

Comment:
do these still belong in here after removal of the sequence from the reco part?

Reply:
not needed because they are not produced

Reply:
removed in https://github.com/cms-sw/cmssw/pull/19893/commits/a81f71edbe4501f3aae02326a93fef59d3ec7dc6